hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieIndexer.java

```diff
@@ -63,6 +63,7 @@
 import static org.apache.hudi.utilities.UtilHelpers.SCHEDULE_AND_EXECUTE;
 
 /**
+ * TODO: [HUDI-8086]
```

Comment:
We can have a separate Jira to update all Javadocs containing Spark examples.

Reply:
https://issues.apache.org/jira/browse/HUDI-8294